src/pages/CrimeInfo/CrimeInfo.js

```diff
@@ -0,0 +1,35 @@
+import React, { useState, useEffect } from "react";
+import axios from "axios";
+
+export default function CrimeInfo(props) {
+  const [info, setInfo] = useState([]);
+
+  const API_KEY = "e14y0OqlWGddCHCXLC3UzLBlcKecsGrxlahVvogx";
```

Comment:
Better not to push api keys in the repository itself. We can achieve this with a devDependency called dotenv (we can discuss it later today if you want)

Reply:
Agree! I have looked into the dotenv package and was able to use it to hide the API.